src/components/refactor/rhf-inputs/radio-input.js

```diff
@@ -0,0 +1,43 @@
+import FormControl from '@mui/material/FormControl';
+import { FormControlLabel, FormLabel, Radio, RadioGroup } from '@mui/material';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useController } from 'react-hook-form';
+
+const RadioInput = ({ name, label, id, defaultValue, possibleValues }) => {
```

Comment:
```suggestion
const RadioInput = ({ name, label, id, possibleValues }) => {
```
React hook form is handling the default value, we should not override this behaviour